client/lib/application/src/application-list/application-list.hbs

```diff
@@ -13,4 +26,4 @@
     </ul>
 {{else}}
     No applications.
-{{/if}}
\ No newline at end of file
+{{/if}}
```

Comment:
Also, since you’re already touching the application list: It would be GREAT if we could separate own and other applications.